sdk/src/wallet/operations/balance.rs

```diff
@@ -338,13 +346,18 @@ impl<S: 'static + SecretManage> Wallet<S> {
 
         #[cfg(not(feature = "participation"))]
         {
-            balance.base_coin.available = balance.base_coin.total.saturating_sub(locked_amount);
+            balance.base_coin.available = balance
+                .base_coin
+                .total
+                .saturating_sub(delegation_implicit_accounts_amount)
```

Comment:
Why not just not add it above in the first place?

Reply:
Because it's still part of the total balance

Reply:
Ah, I see